doc/08-Modules.md

```diff
@@ -0,0 +1,68 @@
+# Modules
+
+## Installation
+
+A module should be installed in one of the [configured module paths](03-Configuration.md#general-configuration).
+The default path in most installations is `/usr/share/icingaweb2/modules`.
+
+Each directory in there contains the files for a particular module. The directory's name has to be the one
```

Comment:
An example would be good here for this case.

Reply:
An example for a directory with files in it?

Reply:
"So if you want to install the reporting module, it should be installed under `usr/share/icingaweb2/modules/reporting`".

Reply:
Hm, none of the content goes that directly into detail about a specific example. Did it like so: `icingaweb2-module-map` turns into `/usr/share/icingaweb2/modules/map`